src/vue/modules/movements-history/wrappers/effect.js

```diff
@@ -22,39 +22,39 @@ export class FundedEffect extends BalanceChangedEffect {}
 /**
  * @class LockedEffect
  *
- * Represents effect that is triggered when funds on balance were moved from
+ * Represents effect that is triggered when sales on balance were moved from
  * `available` to `locked`
  */
 export class LockedEffect extends BalanceChangedEffect {}
 
 /**
  * @class IssuedEffect
  *
- * Represents effect that is triggered when balance become funded by issuing
+ * ё
```

Comment:
Yo

Reply:
Make a review of your PR on GitHub before create one so I did not find such nonsense in future